src/main/kotlin/fi/metatavu/timebank/api/forecast/ForecastService.kt

```diff
@@ -52,4 +52,25 @@ class ForecastService {
         return doRequest("/v2/persons")
     }
 
+    /**
+     * Gets time registrations from Forecast
+     *
+     * @OptionalParam date after in YYYY-MM-DD LocalDate
+     * @return Response with time registrations data
+     */
+    fun getTimeEntries(after: LocalDate?, pageNumber: Int): String? {
+        var path = "/v4/time_registrations"
+        if (after != null) path += "/updated_after/${after.toString().replace("-", "")}T000000"
+        path = "$path?pageSize=1000&pageNumber=$pageNumber"
+        return doRequest(path)
```

Comment:
I'd do this something like this:
```
        val pathSections = mutableListOf<String>()
        pathSections.add("/v4/time_registrations")
        if (after != null) {
            pathSections.add("/updated_after/${after.toString().replace("-", "")}T000000")
        }

        pathSections.add("?pageSize=1000&pageNumber=$pageNumber")
        return doRequest(pathSections.joinToString(""))
```

Instead of concatenate string it makes more sense to construct a list of path sections and join those to a single string at the end. In the optimal world we would have api spec file and generated client that would handle the rest calls